flocker/provision/_install.py

```diff
@@ -58,6 +58,18 @@ def is_centos(distribution):
     return distribution.startswith("centos-")
 
 
+def is_ubuntu(distribution):
+    """
+    Determine whether the named distribution is a version of Ubuntu.
+
+    :param bytes distribution: The name of the distribution to inspect.
+
+    :return: ``True`` if the distribution named is a version of Ubuntu,
+        ``False`` otherwise.
+    """
+    return distribution.startswith("ubuntu-")
+
+
```

Comment:
No action required. 

I mentioned on a previous review that having an `is_centos` function and a bunch of methods with `centos` in the name was a hint that maybe there should be an object. This change makes that more true. However, out of scope for this refactoring.